rowan/components/Dependent-SUnit-Extensions.ston

```diff
@@ -1,7 +1,10 @@
 RwSimpleProjectLoadComponentV2 {
   #name : 'Dependent-SUnit-Extensions',
   #condition : 'sunit',
-  #projectNames : [ ],
+  #projectNames : [
+    'Bell',
+    'Regex'
+  ],
```

Comment:
I think this is not needed now. Bell is already a dependency of the `Deployment` component , that's already included here. And Regex is now a transitive dependency.